lib/jekyll/utils/rouge.rb

```diff
@@ -4,16 +4,16 @@ module Rouge
 
       def self.html_formatter(*args)
         Jekyll::External.require_with_graceful_fail("rouge")
-        html_formatter = ::Rouge::Formatters::HTML.new(*args)
-        return html_formatter if old_api?
-
-        ::Rouge::Formatters::HTMLPygments.new(html_formatter)
+        if old_api?
+          ::Rouge::Formatters::HTML.new(*args)
+        else
+          ::Rouge::Formatters::HTMLLegacy.new(*args)
```

Comment:
Yes, exactly. It produces (almost) the same output as rouge 1.11 while receiving the same arguments.